File: Special_Array_With_X_Elements_Greater_Than_or_Equal_X.py
nums = [3,5]
def specialArray(nums):
    n = max(nums)
    for i in range(n+1):
        count = 0
        for j in nums:
            if j >= i:
                count +=1

        if (count == i):
            return count

    return -1


print(specialArray(nums))

File: Special_Array_With_X_Elements_Greater_Than_or_Equal_X.go
package main

import (
	"fmt"
)

func main() {
	nums := []int{3, 5}
	fmt.Println(specialArray(nums))
}

func specialArray(nums []int) int {
	n := len(nums) + 1

	for i := 1; i <= n; i++ {
		var count int
		for _, j := range nums {
			if j >= i {
				count++
			}
		}
		if count == i {
			return count
		}
	}
	return -1
}
